OpenSim/Simulation/Test/testStatesTrajectory.cpp

```diff
@@ -179,13 +179,14 @@ void testFromStatesStorageGivesCorrectStates() {
 
     Storage sto(statesStoFname);
 
-    // This will fail because we have not yet called initSystem() on the model.
-    SimTK_TEST_MUST_THROW_EXC(
-            StatesTrajectory::createFromStatesStorage(model, sto),
-            OpenSim::ModelHasNoSystem);
+    // It's important that we have not yet initialized the model,
+    // since `createFromStatesStorage()` should be able to work with such a
+    // model.
```

Comment:
This comment is confusing to me. You just want o verify that you can load state trajectories without invoking `model.initSystem()` yourself.